src/extra/gpkg/write.jl

```diff
@@ -0,0 +1,294 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+function gpkgwrite(fname, geotable;)
+  db = SQLite.DB(fname)
+
+  DBInterface.execute(db, "PRAGMA synchronous=0")
+  # Commits can be orders of magnitude faster with
+  # Setting PRAGMA synchronous=OFF but,
+  # can cause the database to go corrupt
+  # if there is an operating-system crash or power failure.
+  # If the power never goes out and no programs ever crash
+  # on you system then Synchronous = OFF is for you
+
+  SQLite.transaction(db) do
```

Comment:
I am noticing that sometimes we use `DBInterface` and sometimes `SQLite`. Should we pick one and stick to it? Perhaps `SQLite` is the more specific one that we need?